.env.sample

```diff
@@ -4,3 +4,16 @@ DISCORD_GUILD_ID=xxxxxxx
 DISCORD_CLIENT_ID=xxxxx
 
 REDIS_URL=redis://localhost:6379
+
+# Channels
+# You can override the ID of any channel specified in the config file by setting it here
+# The format is CHANNEL_NAME_CHANNEL_ID=CHANNEL_ID, e.g. INTRODUCTIONS_CHANNEL_ID=123456789
+# For keys that expect an array, set the value as a comma-separated list of IDs, e.g. NO_POINTS_CHANNEL_IDs=123456789,987654321
+NO_POINTS_CHANNEL_IDs='',''
```

Comment:
Don't want it to break :P 
+ avoiding "expect an array" in case someone actually does enter `"[32498327423]"`
```suggestion
# For keys that accept multiple channel IDs, set the value as a comma-separated list of IDs, e.g. NO_POINTS_CHANNEL_IDs=123456789,987654321
NO_POINTS_CHANNEL_IDs='' # accepts multiple channel IDs
```